src/services/analytics/events/txList.ts

```diff
@@ -54,6 +54,12 @@ export const TX_LIST_EVENTS = {
     category: TX_LIST_CATEGORY,
     event: EventType.CLICK,
   },
+  TOGGLE_UNTRUSTED: {
+    action: 'Toggle untrusted transactions',
+    category: TX_LIST_CATEGORY,
+    event: EventType.CLICK,
+    // label: 'hide' | 'show',
```

Comment:
Can we remove this?

Reply:
Why? It's supposed to be a piece of documentation :)
We have the same line in a few other events.

Reply:
ahh okay. Now I understand the comment 💡 
Nevermind, it's late 😪